unittests/InstructionCountCI/RPRES/VEX_map1_AFP.json

```diff
@@ -29,14 +29,15 @@
       ]
     },
     "vrsqrtss xmm0, xmm1, xmm2": {
-      "ExpectedInstructionCount": 2,
+      "ExpectedInstructionCount": 3,
       "Comment": [
         "AFP can make this more optimal",
         "Map 1 0b10 0x52 128-bit"
       ],
       "ExpectedArm64ASM": [
         "mov v16.16b, v17.16b",
-        "frsqrte s16, s18"
+        "frsqrte s0, s18",
```

Comment:
Following from the previous comment about `vrsqrtss` with only AFP and not rpres. Theoretically the original version of this should have correct semantics.
We are first moving xmm1 in to xmm0
We then `frsqrte s16 (xmm0), s18 (xmm2)` and following the pseudocode:
```
boolean merge = elements == 1 && IsMerging(FPCR);
bits(128) result = if merge then V[d, 128] else Zeros(128);
```
vd is used as the merge source, which should follow semantics correctly.
Same thing about the `vrcpss` down below.